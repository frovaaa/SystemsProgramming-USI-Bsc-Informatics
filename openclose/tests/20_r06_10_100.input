502b0d
97149
bd4367
40574
81259
24971
90230
3965bd
76476
83282
9837bd
pq5b30d
45777
17286
48871
54965
19953
87424
b623pqd
0462pq
34451
p9670q
15192
4p845q
83449
19438
42733
44p27q
45551
68791
39508
4bd6p8q
67778
81927
37390
1p947q
pq5050
0b296d
98580
79858
764b4d
17p8q0
99375
428b8d
676b0d
25580
35005
83849
8898bd
8b447d
5pp16qq
53157
90102
24835
64310
p0412q
98332
73891
263bd8
08340
25879
92824
1bp20qd
56122
p1073q
74pq33
p5058q
91b9pqd
bd7b80d
5b9d10
47624
43413
74474
33354
5171pq
84404
4p6pbdqq
12674
8p238q
35531
66518
4p457q
p2669q
8pq452
39p6q3
91486
95606
255p2q
59909
58909
40074
94704
10441
00281
90364
b5930d
81333
1080pq
5p91pqq
52bd36
