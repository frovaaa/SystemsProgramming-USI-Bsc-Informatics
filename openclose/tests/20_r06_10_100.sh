#!/bin/bash
exec "$1" "bdpq" < tests/20_r06_10_100.input
