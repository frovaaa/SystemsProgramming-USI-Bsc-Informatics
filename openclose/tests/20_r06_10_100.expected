502b0d
97149
4367
40574
81259
24971
90230
3965
76476
83282
9837
5b30d
45777
17286
48871
54965
19953
87424
b623d
0462
34451
p9670q
15192
4p845q
83449
19438
42733
44p27q
45551
68791
39508
46p8q
67778
81927
37390
1p947q
5050
0b296d
98580
79858
764b4d
17p8q0
99375
428b8d
676b0d
25580
35005
83849
8898
8b447d
5pp16qq
53157
90102
24835
64310
p0412q
98332
73891
2638
08340
25879
92824
1bp20qd
56122
p1073q
7433
p5058q
91b9d
7b80d
5b9d10
47624
43413
74474
33354
5171
84404
4p6q
12674
8p238q
35531
66518
4p457q
p2669q
8452
39p6q3
91486
95606
255p2q
59909
58909
40074
94704
10441
00281
90364
b5930d
81333
1080
5p91q
5236
